sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2671,6 +2671,25 @@ object SQLConf {
       .checkValue(_ > 0, "The difference must be positive.")
       .createWithDefault(4)
 
+  val NULL_AWARE_ANTI_JOIN_OPTIMIZE_ENABLED =
```

Comment:
Yes. it's only for review and testing.
I think we need to make a decision on whether BroadcastHashJoinExec or BroadcastNullAwareHashJoinExec is used?

no matter what, NULL_AWARE_ANTI_JOIN_OPTIMIZE_ENABLED this will be the only switch left in final submit.